Jellyfin.Api/Controllers/StartupController.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System.Linq;
+using System.Threading.Tasks;
+using MediaBrowser.Controller.Configuration;
+using MediaBrowser.Controller.Library;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Jellyfin.Api.Controllers
+{
+    [Route("emby/[controller]")]
```

Comment:
Can we make this a variable? Specifically the `emby` part - better to get in this habit now so we can drop in a `baseUrl` later.

Reply:
This can wait for future developments, since it's certainly not the only location.